### ISSUE TITLE
How to play the byte array received through media player?

### ISSUE THREAD

**Issue by ganeshw27** [SteeBono/airplayreceiver#19]
How to programmatically play the video packets received in byte array through VLC or Media Element? I captured the 10-minute feed and later played it manually through the VLC player but the video saved was only 2 mins in very fast forwarding mode and some parts missing. So how to resolve it?

**Comment by indroidios** [comment]
Yes, but if I want to play the live feed from the iPhone then how will it work? I think it's good for storing and playing but for live, we need to play it without storing it on the disk.

**Comment by SteeBono** [comment]
Yes, you can play live feed from iPhone.
This source code allows you to decode the audio and video stream.
It is then up to you to develop the code to process the stream in real time to VLC or any video / audio player.
